.github/workflows/doctest.yaml

```diff
@@ -13,25 +13,27 @@ jobs:
                'doc/starterkit/k4MarlinWrapperCLIC/CEDViaWrapper']
     steps:
     - uses: actions/checkout@v3
-    - uses: cvmfs-contrib/github-action-cvmfs@v3.1
     - name: Start container
       run: |
-        docker run -it --name CI_container -v ${GITHUB_WORKSPACE}:/Package -v /cvmfs:/cvmfs:shared -d ghcr.io/aidasoft/centos7:latest /bin/bash
+        docker run -it --name CI_container --privileged -v ${GITHUB_WORKSPACE}:/Package -d ghcr.io/key4hep/key4hep-images/alma9-cvmfs:latest /bin/bash
     - name: Compile
       run: |
-        docker exec CI_container /bin/bash -c ' cd Package;/
-        source ${{ matrix.SETUP }};\
-        mkdir build; cd build;\
-        cmake .. -GNinja -DCMAKE_INSTALL_PREFIX=../install;\
-        ninja install;\
+        docker exec CI_container /bin/bash -c '/mount.sh;
+        cd Package;
+        source ${{ matrix.SETUP }};
+        mkdir build; cd build;
+        cmake .. -GNinja -DCMAKE_INSTALL_PREFIX=../install;
+        ninja install;
         '
     - name: CheckPage
       run: |
-        docker exec CI_container /bin/bash -c 'cd ./Package;
+        docker exec CI_container /bin/bash -c '/mount.sh
+        cd ./Package;
         source ${{ matrix.SETUP }};
         export LD_LIBRARY_PATH=/Package/install/lib64:$LD_LIBRARY_PATH
         export PYTHONPATH=/Package/install/python:$PYTHONPATH
         export K4MARLINWRAPPER=/Package/install/share/k4MarlinWrapper
+        unset KEY4HEP_STACK;
```

Comment:
@jmcarcell this is rather brute-force but it should work. Should we think about a flag to the setup script that simply doesn't do anything if a key4hep stack is already setup?